ckeditor_blocks.module

```diff
@@ -155,15 +171,16 @@ function ckeditor_blocks_block_ajax($module, $delta) {
 function ckeditor_blocks_ckeditor_plugins() {
   $path = backdrop_get_path('module', 'ckeditor_blocks') . '/plugins/blocks';
   $plugins['blocks'] = array(
-      'path' => $path,
-      'file' => 'plugin.js',
-      'internal' => FALSE,
-      'buttons' => array(
-          'Blocks' => array(
-              'label' => t('Blocks'),
-              'image' => $path . '/icons/blocks.png',
-          ),
+    'path' => $path,
+    'file' => 'plugin.js',
+    'css' => array(backdrop_get_path('module', 'ckeditor_blocks') . '/css/ckeditor_blocks.css'),
```

Comment:
we already call backdrop_get_path() above, can we call it only once and tack on the two different endings? micro-efficiency improvement :)

Reply:
Yes, of course! What happened is that in this branch I didn't have that call to backdrop_get_path() which was introduced in a different PR. When resolving conflicts it got there and I didn't see the redundancy. Good catch, thanks! I'll fix it